components/Home/Home.jsx

```diff
@@ -128,37 +128,36 @@ export default () => {
   }, []);
 
   const welcomeName = firebase.auth().currentUser.displayName;
-  
+
   const workouts = useSelector((state) => state.workouts.workouts);
   const cycles = useSelector((state) => state.cycles);
   const dispatch = useDispatch();
 
   // Parse the database response into workoutList
   const workoutList = workouts.map((workout) => ({
     name: workout.name,
-    lastPerformed : workout.lastPerformed,
+    lastPerformed: workout.lastPerformed,
     subtext: `${workout.lastPerformed} days ago`,
     id: workout.id,
     color: workout.color,
   }));
-  
+
   // Filter the workout list to show only the 5 most recently performed workouts
   // If less than 5, display all workouts
   const filterWorkoutListForDisplay = (workoutList) => {
-      workoutList.sort(function(a,b) {
-        if (isNaN(a.lastPerformed)) return 1;
-        if (isNaN(b.lastPerformed)) return -1;
-        if (a.lastPerformed == b.lastPerformed) return 0;
-        return (a.lastPerformed > b.lastPerformed ? 1 : -1)
+    workoutList.sort((a, b) => {
+      if (isNaN(a.lastPerformed)) return 1;
+      if (isNaN(b.lastPerformed)) return -1;
+      if (a.lastPerformed == b.lastPerformed) return 0;
+      return (a.lastPerformed > b.lastPerformed ? 1 : -1);
     });
     workoutList.forEach((workout) => {
       if (isNaN(workout.lastPerformed)) {
-        workout.subtext = "Try for first time!";
+        workout.subtext = 'Try for first time!';
       }
     });
-    return workoutList.slice(0,5);
-  }
-
+    return workoutList.slice(0, 5);
+  };
 
   const selectedCycle = (cycles.selectedCycleId !== undefined) && _.find(cycles.cycles, (cycle) => cycle.id === cycles.selectedCycleId);
   let cycleDetails;
```

Comment:
You will get an index undefined error if you try to click on a left or right arrow on the home page if a cycle isn't set

Reply:
> You will get an index undefined error if you try to click on a left or right arrow on the home page if a cycle isn't set

Im also getting the same error if I try to create a cycle while not having a cycle currently selected (new account).